### PR TITLE
new task status will be returned

### PR DESCRIPTION
**Pull request by akshay1502**
Updating task status to the new task workflow while returning.

### CODE REVIEW

**Review by Pavangbhat — CHANGES_REQUESTED** [review]
Are we updating tasks with new task workflow status ? i.e
**active** → **IN_PROGRESS**, **unAssigned** → **AVAILABLE**

And also please note that the task is considered as active if it's in **BLOCKED**, **IN_PROGRESS**, **SMOKE_TESTING**

for reference ⬇️ 
image

**Review by prakashchoudhary07 — APPROVED** [review]
looks good to me :+1:

File: models/tasks.js [hunk 1]
```diff
@@ -86,8 +90,11 @@ const fetchActiveTaskMembers = async () => {
 const fetchTask = async (taskId) => {
   try {
     const task = await tasksModel.doc(taskId).get()
-    const taskData = task.data()
-    return { taskData: await fromFirestoreData(taskData) }
+    const taskData = await fromFirestoreData(task.data())
+    if (taskData?.status) {
+      taskData.status = TASK_STATUS[taskData.status]
```
Comment (Pavangbhat) [inline models/tasks.js:95]: Currently, we have task status as **pending** and **High Priority** please those cases as well

**Review by Pavangbhat — CHANGES_REQUESTED** [review]
Hi, Akshay great work  🎉 
There are a few changes needed please work on it.

Also, make sure the following scenarios are handled 

- [ ] we can hit **/tasks/username?status=active** where we need to return tasks that are in either of the status **[active, blocked, completed, pending, IN_PROGRESS, BLOCKED,  SMOKE_TESTING, COMPLETED]**

- [ ] idle members are those who don't have a task which is either in [active, blocked, pending, IN_PROGRESS, BLOCKED,  SMOKE_TESTING]

**Comment by Pavangbhat** [comment]
This PR is continued with #514

**Pull request merged**